File: Array/Prob02/Prob2.cpp
#include <iostream>
using std::cout;

int main() {
    int arr[] = {10, -9, 20, 11, 100, 210, 49, -3};
    int len = sizeof(arr)/sizeof(arr[0]);

    int min, max;
    min = max = arr[0];

    for (int i = 0; i < len; i++) {
        if (arr[i] > max) max = arr[i];
        if (arr[i] < min) min = arr[i];
    }

    cout << "Max: " << max << "\nMin: " << min;
    return 0;
}

// Time complexity: O(n)  [n = size of array]

File: Array/Prob02/Prob2.java
public class Prob2 {
    public static void main(String[] args) {
        int[] arr = {10, -9, 20, 11, 100, 210, 49, -3};
        int max = arr[0];
        int min = arr[0];

        for (int i = 0; i < arr.length; i++) {
            if (arr[i] > max) max = arr[i];
            if (arr[i] < min) min = arr[i];
        }

        System.out.println("Max: " + max + "\nMin: " + min);
    }
}
